app/init.php

```diff
@@ -1329,18 +1332,7 @@ function getDevice($root): Device
 
 App::setResource('sms', function () {
     $dsn = new DSN(App::getEnv('_APP_SMS_PROVIDER'));
-    $user = $dsn->getUser();
-    $secret = $dsn->getPassword();
-
-    return match ($dsn->getHost()) {
-        'mock' => new Mock($user, $secret), // used for tests
-        'twilio' => new Twilio($user, $secret),
-        'text-magic' => new TextMagic($user, $secret),
-        'telesign' => new Telesign($user, $secret),
-        'msg91' => new Msg91($user, $secret),
-        'vonage' => new Vonage($user, $secret),
-        default => null
-    };
+    return SMSFactory::createFromDSN($dsn);
 });
```

Comment:
Do we actually inject this resource anywhere? If not, let's remove it

Reply:
Appears not. Will remove it